content/c-sharp/concepts/math-functions/terms/tan/tan.md

```diff
@@ -0,0 +1,67 @@
+---
+Title: '.Tan()'
+Description: 'Returns the tangent of a given angle.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Numbers'
+  - 'Arithmetic'
+  - 'Functions' 
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+In C#, the **`Math.Tan()`** method returns the tangent of a given angle.
+
+## Syntax
+
+```pseudo
+Math.Tan(angle);
+```
+
+The `Math.Tan()` method takes only one parameter, `angle`, an angle in radians of type `double`. The method returns the tangent of the `angle` as a `double` value or `NaN` (not a number) if the value of `angle` equals:
+
+- `NaN`
+- `NegativeInfinity`
+- `PositiveInfinity`
+
+## Example
```

Comment:
Generally I recommend having only one example for simple methods. I would suggest deleting one of these, since they have the same code, and there are not many parameters that can be featured in a second example.

Reply:
So, should I simplify the example above? like this

```
double sine = Math.Tan(RadianAngle);
```

instead of deleting it? Because, in the issue #1616 description, both `cs example` and `runable codebyte` is needed, so I don't think we can detete one of these.

Reply:
Having both is not a requirement, it is a convention we typically follow. Feel free to modify one example or delete it.

Reply:
@caupolicandiaz  okay, I've just modified example, can you check it again?